__tests__/app.test.js

```diff
@@ -32,3 +32,31 @@ describe('GET /api/topics' , () => {
     });
 })
 
+describe('GET /api/articles', () => {
+  test('responds with an array of objects, with author, title, article-id, topic, created-at, votes, and comment count keys', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({body}) => {
+      body.articles.forEach((article) => {
```

Comment:
When using a .forEach loop in tests, just make sure you also check the array length. If the returned array were empty, this test would still pass as the .forEach would be skipped.